src/corefx/System.Globalization.Native/timeZoneInfo.cpp

```diff
@@ -18,11 +18,12 @@ extern "C" int32_t GlobalizationNative_ReadLink(const char* path, char* result,
 {
```

Comment:
You can delete `Globalization_ReadLink` and just use `SystemNative_ReadLink`. `Globalization_ReadLink` is left over from times when we have not used `SystemNative_...` in CoreLib.

Reply:
https://github.com/dotnet/corefx/blob/master/src/Common/src/Interop/Unix/System.Native/Interop.ReadLink.cs